at_commons/test/at_key_type_test.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:at_commons/at_commons.dart';
+import 'package:at_commons/src/keystore/key_type.dart';
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
Should the tests be extended for various reserved keys?

Reply:
yes Jagan..I will take colin's prod hive box and run through all the keys and write tests to cover the missed scenarios